src/test/HostActivationTests/GivenThatICareAboutMultilevelSharedFxLookup.cs

```diff
@@ -486,10 +702,101 @@ private void SetRuntimeConfigJson(string destFile, string version, int? rollFwdO
+            dir.Create();
+
+            string fxName = "UberFx";
+            string testAssembly = "System.Collections.Immutable";
+
+            // Create the deps.json. Generated file:
+            /*
+                {
+                  "runtimeTarget": {
+                    "name": "UberFx"
+                  },
+                  "targets": {
+                    "UberFx": {
+                      "System.Collections.Immutable": {
+                        "runtime": {
+                          "System.Collections.Immutable.dll": {}
+                        }
+                      }
+                    }
+                  },
+                  "libraries": {
+                    "System.Collections.Immutable": {
+                      "type": "assemblyreference",
+                      "serviceable": false,
+                      "sha512": ""
+                    }
+                  }
+                }
+             */
+            JObject depsjson = new JObject(
+                new JProperty("runtimeTarget",
+                    new JObject(
+                        new JProperty("name", fxName)
+                    )
+                ),
+                new JProperty("targets",
+                    new JObject(
+                      new JProperty(fxName,
+                          new JObject(
+                              new JProperty(testAssembly,
+                                  new JObject(
+                                      new JProperty("runtime",
+                                          new JObject(
+                                              new JProperty(testAssembly + ".dll",
+                                                  new JObject()
+                                              )
+                                          )
+                                      )
+                                  )
+                              )
+                          )
+                      )
+                  )
+              ),
+                  new JProperty("libraries",
+                      new JObject(
+                          new JProperty(testAssembly,
+                            new JObject(
+                                new JProperty("type", "assemblyreference"),
+                                new JProperty("serviceable", false),
+                                new JProperty("sha512", "")
+                            )
+                        )
+                    )
+                )
+            );
+
+            string depsFile = Path.Combine(builtSharedUberFxDir, "Microsoft.UberFramework.deps.json");
+
+            File.WriteAllText(depsFile, depsjson.ToString());
+
+            // Copy the test assembly
+            string fileSource = Path.Combine(builtSharedFxDir, testAssembly + ".dll");
+            string fileDest = Path.Combine(builtSharedUberFxDir, testAssembly + ".dll");
+            FileInfo fileInfo = new FileInfo(fileSource);
+            fileInfo.CopyTo(fileDest);
```

Comment:
I don't think you need to use `FileInfo` here.  Can't you just say:

`File.Copy(fileSource, fileDest)`

Reply:
OK